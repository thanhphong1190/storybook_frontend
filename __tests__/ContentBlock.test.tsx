import {
  ContentBlock,
  ContentBlocksProps
} from '../src/components/blocks/ContentBlock'
import renderer from 'react-test-renderer'

const props: ContentBlocksProps = {
  headerText: `Who are plus fitness`,
  contentPrimayText: `Plus Fitness is an Australian owned franchise offering affordable and flexible gym memberships at over 180 convenient locations across Australia and New Zealand.
  <br /><br />
  By joining one Plus Fitness you can instantly use all of our gyms and with no lock in contracts, 
  24 hour access*, free membership time-hold, great classes, free fitness programs and ongoing support, 
  affordable Personal Training plus our ever popular Functional Training Zones, there is nothing else you could want in a gym!`,
  contentSecondaryText: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.`,
  primaryBtnText: 'JOIN NOW',
  secondaryBtnText: 'ENQUIRE NOW'
}

it('renders correctly', () => {
  const tree = renderer.create(<ContentBlock {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
