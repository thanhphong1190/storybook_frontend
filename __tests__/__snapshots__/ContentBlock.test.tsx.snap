// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="globalstyles__Card-sc-1nc69wi-3 ktPNbg"
>
  <h2
    className="globalstyles__CardHeader-sc-1nc69wi-4 kzlczZ"
  >
    Who are plus fitness
  </h2>
  <div
    className="globalstyles__CardBody-sc-1nc69wi-5 fqKNQF"
  >
    <p
      className="ContentBlockstyled__ContentBlockText-qi3y14-0 dezrkY"
      color="orange"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Plus Fitness is an Australian owned franchise offering affordable and flexible gym memberships at over 180 convenient locations across Australia and New Zealand.
  <br /><br />
  By joining one Plus Fitness you can instantly use all of our gyms and with no lock in contracts, 
  24 hour access*, free membership time-hold, great classes, free fitness programs and ongoing support, 
  affordable Personal Training plus our ever popular Functional Training Zones, there is nothing else you could want in a gym!",
        }
      }
    />
    <p
      className="ContentBlockstyled__ContentBlockText-qi3y14-0 eytvUc"
      color="black"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.",
        }
      }
    />
  </div>
  <div
    className="globalstyles__CardFooter-sc-1nc69wi-6 cnCxTF"
  >
    <button
      className="Button-lxhj0a-0 djWYEe"
      color="blue"
      onClick={[Function]}
    >
      ENQUIRE NOW
    </button>
    <button
      className="Button-lxhj0a-0 eQpnsG"
      onClick={[Function]}
    >
      JOIN NOW
    </button>
  </div>
</div>
`;
